src/pending_tasks/api/serializers.py

```diff
@@ -0,0 +1,46 @@
+from rest_framework import serializers
+
+from auth_and_perms.models import Profile, Rol
+from pending_tasks.models import PendingTask
+from djgentelella.widgets import core as genwidgets
+
+
+class PendingTaskSerializer(serializers.ModelSerializer):
+    status = serializers.SerializerMethodField()
+    status_display = serializers.SerializerMethodField()
+    validate_profile = serializers.SerializerMethodField()
+    rols = serializers.SerializerMethodField()
+
+    def get_status(self, obj):
+        return {'id': obj.status, 'name': self.get_status_display(obj)}
+
+    def get_status_display(self, obj):
+        return obj.get_status_display()
+
+    def get_validate_profile(self, obj):
+        if obj.validate_profile:
```

Comment:
If profile is null, you should return something like None, but we recommend you using the GTS2SerializerBase class, this class is defined in djgentelella package. So you can defined validated_profile = GTS2SerializerBase() and the return method is not neccesary in this case.